Modules/IO/XML/include/itkDOMNode.h

```diff
@@ -93,11 +93,11 @@ class ITKIOXML_EXPORT DOMNode : public Object
   GetParent() const;
 
   /** Retrieve the tag name of this node. */
-  itkSetMacro(Name, std::string &);
+  itkSetMacro(Name, const std::string &);
```

Comment:
These are the only use cases for of `itkSetMacro\.\*\&`. Wouldn't it now make more sense to just pass by value and not reference?

Reply:
@blowekamp I think so, yes: pass by value usually seems to be the right thing to do, for a `Set` member function that passes a "regular" copyable argument.

This appears to be a rare case where itkSetMacro was used to pass by const-reference. I just left it that way, to avoid breaking the ABI unnecessarily. However, it could be changed to pass by value in a follow-up commit, if anyone wants to.

Some motivation: historically, in C++ an input argument of a class types was usually passed by const-reference, to avoid copying. However, with a Set member functions, the argument might as well be passed by value, as its value can now be "moved" into the corresponding data member.

Reply:
I would love to see a followup request made with a ITK v 6.0.0 release tag added.

Hans